Sources/SwiftFrameCore/Workers/CommandParser.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+import SPMUtility
+import Yams
+
+public class CommandParser {
+
+    public struct Result {
+
+        public enum ConfigFormat {
+            case json
+            case yaml
+        }
+
+        public let verbose: Bool
+        public let path: String
+        public let format: ConfigFormat
+
+    }
+
+    private let parser: ArgumentParser
+    private let configFlag: OptionArgument<String>
+    private let verboseFlag: OptionArgument<Bool>
+    private let configFormatFlag: OptionArgument<String>
+
+    public init() {
+        parser = ArgumentParser(
+            commandName: nil,
+            usage: "[-c path] [-v] [-f format]",
+            overview: "Overlay screenshots from your app with a template file and text and render them",
+            seeAlso: nil)
+        configFlag = parser.add(
+            option: "--config",
+            shortName: "-c",
+            kind: String.self,
+            usage: "Read configuration values from the specified file",
+            completion: .filename)
+        verboseFlag = parser.add(
+            option: "--verbose",
+            shortName: "-v",
+            kind: Bool.self,
+            usage: "Prints additional information and lets you verify the config file before rendering",
+            completion: .unspecified)
+        configFormatFlag = parser.add(
+            option: "--format",
+            shortName: "-f",
+            kind: String.self,
+            usage: "Specifies the format of the config file. \"json\" or \"yaml\" are allowed",
```

Comment:
I wonder if we could auto-infer this flag. E.g. you could require a file ending (`.json` or `.yaml`) on the config file. Alternatively, you could inspect the config file and if it doesn't start with `{`, you could assume that it's YAML. Less options to specify always makes a tool easier to use. :)